students/shumin/lesson02/print_grid.py

```diff
@@ -0,0 +1,13 @@
+# This program prints grid
+# The size of the grid is controlled by the parameter
+
+def print_grid(n):
+    numSides = n // 2
+    plus = "+ "
+    minus = "- "
+    empty = "  "
+    pipe = "| "
+    NextLine = "\n"
+    HoriLine = plus + minus * numSides + plus + minus * numSides + plus + NextLine
+    VertLine = (pipe + empty * numSides + pipe + empty * numSides + pipe + NextLine) * numSides
```

Comment:
make sure to follow pep8 guidelines: variable names should use snake_case style -> 'vert_line` (or `vertical_line`)